services/tasks/local.go

```diff
@@ -266,12 +266,18 @@ func (l *local) DeleteProcess(ctx context.Context, r *api.DeleteProcessRequest,
 	}, nil
 }
 
-func processFromContainerd(ctx context.Context, p runtime.Process) (*task.Process, error) {
+func getProcessState(ctx context.Context, p runtime.Process) (*task.Process, error) {
+	ctx, cancel := context.WithTimeout(ctx, 2*time.Second)
+	defer cancel()
+
 	state, err := p.State(ctx)
 	if err != nil {
-		return nil, err
+		if errdefs.IsNotFound(err) {
+			return nil, err
+		}
+		log.G(ctx).WithError(err).Errorf("get state for %s", p.ID())
```

Comment:
I don't get this part. Why would skip returning an error here?

Reply:
We still know the task exists but we are unable to fetch its state.  We should still return the task as it does exist but return in an unknown state.  This way the use can make a delete call if it is stuck and fix the situation.

Reply:
I see. Ok cool thanks.